core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -155,12 +195,64 @@ oppia.controller('StatisticsTab', [
           controller: [
             '$scope', '$modalInstance', '$filter', 'stateName', 'stateStats',
             'improvementType', 'visualizationsInfo', 'HtmlEscaperService',
+            'ENABLE_NEW_STATS_FRAMEWORK',
             function($scope, $modalInstance, $filter, stateName, stateStats,
-                improvementType, visualizationsInfo, HtmlEscaperService) {
+                improvementType, visualizationsInfo, HtmlEscaperService,
+                ENABLE_NEW_STATS_FRAMEWORK) {
+              $scope.COMPLETION_RATE_PIE_CHART_OPTIONS1 = {
+                title: 'Answer feedback statistics',
+                left: 0,
+                pieHole: 0.6,
+                pieSliceTextStyleColor: 'black',
+                pieSliceBorderColor: 'black',
+                chartAreaWidth: 240,
+                colors: ['#d8d8d8', '#008808', 'blue'],
+                height: 270,
+                legendPosition: 'right',
+                width: 240
+              };
+              $scope.COMPLETION_RATE_PIE_CHART_OPTIONS2 = {
```

Comment:
Too much duplication; find a way to parametrize.

Reply:
Done.